index.html

```diff
@@ -0,0 +1,102 @@
+    <section class="services">
+        <h2 class="services-title">services</h2>
+        <hr>
+        <ul class="services-wrapper">
+            <li class="services-item">
+              <i class="service-icon fas fa-calculator"></i>
+              <p class="service-title">automatic calculate interest</p>              
+              <p class="service-text">
+                our system gives you the interest that you'll receive automatically.
+              </p>
+            </li>
+            <li class="services-item">
+                <i class="service-icon fas fa-dollar-sign"></i>
+              <p class="service-title">automatic calculate mora</p>              
+              <p class="service-text">
+                if you client doesnt pay at time the system calculates a due interest. 
+              </p>
+            </li>
+            <li class="services-item">
+                <i class="service-icon fas fa-file-alt"></i>
+              <p class="service-title">payments reports</p>              
+              <p class="service-text">
+                you'll be able to see the payments that you clients do in a daily, weekly and monthly basis.
+              </p>
+            </li>
+            <li class="services-item">
+                <i class="service-icon fas fa-print"></i>
+              <p class="service-title">print your reports</p>              
+              <p class="service-text">
+                you'll be able to print the payments reports whenever you need it.
+              </p>
+            </li>
+            <li class="services-item">
+                <i class="service-icon fas fa-sign-in-alt"></i>
+              <p class="service-title">access to your data</p>              
+              <p class="service-text">
+                you'll be able to access to your business data whenever you want and wherever you are.
+              </p>
+            </li>
+        </ul>
+        <button class="service-btn">more info</button>
+    </section>
+    <section class="testimonials">
+        <h2 class="testimonials-title">testimonials</h2>
+        <hr class="separator-testimonials">
+        <ul class="testimonials-wrapper" id='testimonial-section'>
+            
+            
+        </ul>
+        
+          <div class="btn-testimonials" onclick="showAllTestimonials()">
+            <p id="btn-text">more</p><span id="arrow">&#711;</span>
+          </div>
+        
+    </section>
+    <footer>
+        <ul class="footer-wrapper">
+            <li class="footer-item item-1">
+                <p class="footer-logo">My Logo</p>
+            </li>
+            <li class="footer-item item-2"> 
+                <p class="footer-text">2021 Alrights Reserved</p>
+                <p class="footer-text text-2">Lassis enterprises save the rights to sell and reproduce this product
+                     <span class="second-part">any copy sold without permission would be suited</span>
+                </p>
+            </li>
```

Comment:
Change background color here as the original design below
![right4](https://user-images.githubusercontent.com/58376020/124036038-aef24b80-d9fd-11eb-96b2-e01affde90cf.png)